Sources/TensorFlow/Core/LazyTensorContext.swift

```diff
@@ -62,12 +62,22 @@ class LazyTensorOperationsTracker {
 
 struct LazyTensorContext {
     private var operationsTracker = LazyTensorOperationsTracker()
+    private var _shapeTrackingEnabled = true
 
     static private var threadLocalContext: LazyTensorContext {
```

Comment:
Consider renaming to just `local`. It's a convention used in a lot of other Swift APIs.